torch/package/package_exporter.py

```diff
@@ -611,6 +619,7 @@ def close(self):
             self.zip_file.write_record(name, storage.data_ptr(), num_bytes)
```

Comment:
PackageExporter doesn't write out the tensor storages names until the end of the package, this prevents the ScriptModuleSerializer from being able to inspect the PyTorchStreamWriter and determine what storages have been used so far. This is why I kept the `get_storage_id()` and `self._next_storage_id`